mlir-tensorrt/python/requirements-dev.txt

```diff
@@ -13,3 +13,4 @@ nvidia-ml-py
 black
 build
 psutil
+torch
```

Comment:
this should be more specific in terms of features/version. Internally we have 

```
--extra-index-url https://download.pytorch.org/whl/cpu
torch==2.1.0+cpu
```

Reply:
It should be installing cpu version by default. I made it explicit.